pkg/chaosctl/debug/iochaos/iochaos.go

```diff
@@ -47,12 +47,9 @@ func Debug(ctx context.Context, chaos runtime.Object, c *cm.ClientSet, result *c
 	for i := range pods {
 		podName := pods[i].Name
 		podResult := cm.PodResult{Name: podName}
-		err := debugEachPod(ctx, pods[i], daemons[i], ioChaos, c, &podResult)
+		_ = debugEachPod(ctx, pods[i], daemons[i], ioChaos, c, &podResult)
 		result.Pods = append(result.Pods, podResult)
-		if err != nil {
-			// TODO: should follow best-effort pattern
-			return errors.Wrap(err, fmt.Sprintf("failed to debug io chaos for pod %s", podName))
-		}
+		// TODO: V(4) log when err != nil, wait for #1433
```

Comment:
should we delete `wait for #1433` since 1433 is merged

Reply:
I will append logging and finish this TODO in another PR. :)